Calling Cards Blog Post

cmatKhan opened the pull request:
This is the much anticipated Calling Cards blog post.

I also adjusted the README instructions on launching the dev server to more explicitly describe the procedure I followed -- that isn't related to the calling cards blog and might be better done in a different push.

mashehu reviewed (approved):
Great job!
just some smaller things (commit with fixes for the figures is incoming).

cmatKhan commented on README.md:
````diff
@@ -39,7 +39,14 @@ npm install
+Then execute the following command:
 
 ```bash
 npm run dev
````
I removed this commit from this branch

cmatKhan commented on sites/main-site/src/assets/images/blog/callingcards/callingcards_vs_chip.png:
done in cb8fe6a71d67a337cd062541cd9c78f249220492

cmatKhan commented on sites/main-site/src/content/blog/2024/callingcards_intro.mdx:
```diff
@@ -0,0 +1,137 @@
+
+# Calling Cards in Yeast
+
+<div style={{ textAlign: 'center', margin: '20px 0' }}>
```
ah, right...that is much better. Thank you for changing all of those

cmatKhan commented:
@mashehu thank you for your commits and review.

I removed the README update from this branch, updated the background, accepted your commits and made some minor text edits.

I'm failing the playwright check now -- I'm running this on my local and it seems like it is failing on pages other than the one i updated.

mashehu commented:
Yes, I hoped the playwright tests would work better with the new monorepo structure, but we haven't found a good way to handle the multiple deployment previews. The tests can be ignored for now.

mashehu commented on sites/main-site/src/content/blog/2024/callingcards_intro.mdx:
```diff
@@ -0,0 +1,144 @@
+subtitle: 'The nf-core callingcards workflow'
+headerImage: '/assets/images/blog/callingcards/callingcards_metromap.png'
+headerImageAlt: 'Subway diagram of the calingcards pipeline'
+pubDate: 2024-07-10T00:19:00+01:00
```
```suggestion
pubDate: 2024-07-17T00:19:00+01:00
```

mashehu commented:
okay, good to merge whenever you are ready 🙂

FriederikeHanssen commented:
The deploy preview looks off (tested on chrome and safari). Is this anythign to be concerned about @mashehu 
<img width="1579" alt="Screenshot 2024-07-17 at 11 33 42" src="https://github.com/user-attachments/assets/39f27632-c9bc-4085-898c-cf23c0c34ce7">

mashehu commented:
Will be fine once it's merged (bug with the new setup and different sources for components)

mashehu commented:
@cmatKhan I set the `pubDate` now to July 18th midnight. feel free to change that to a date more suitable for you (just want to have the "new blog post" indicator appear for this post as well (stays there for 24h past release)
